mscore/importgtp-gp6.cpp

```diff
@@ -2665,17 +2698,28 @@ int GuitarPro6::readBeatEffects(int, Segment*)
 
 bool GuitarPro6::read(QFile* fp)
       {
-      f      = fp;
-      slides = new QMap<int,int>();
-
+      f = fp;
       previousTempo = -1;
       QByteArray buffer = fp->readAll();
 
       // decompress and read files contained within GPX file
       readGPX(&buffer);
-      delete slides;
 
       return true;
       }
 
+//---------------------------------------------------------
+//   GuitarPro7::read
+//---------------------------------------------------------
+
+bool GuitarPro7::read(QFile* fp)
```

Comment:
I believe it would be better to add Guitar Pro 7-related functions to a separate file rather than mixing them with GP6 ones.